pkg/types/alertmanagertypes/alert.go

```diff
@@ -28,19 +27,22 @@ type (
 	// An alias for the Alert type from the alertmanager package.
 	Alert = types.Alert
 
-	// An alias for the PostableAlert type from the alertmanager package.
-	PostableAlert = models.PostableAlert
-
-	// A slice of PostableAlert.
-	PostableAlerts = []*PostableAlert
-
 	// An alias for the GettableAlert type from the alertmanager package.
 	GettableAlert = models.GettableAlert
 
 	// A slice of GettableAlert.
 	GettableAlerts = models.GettableAlerts
+
+	PostableAlerts []*PostableAlert
 )
 
+// PostableAlert embeds models.PostableAlert and adds custom fields
+type PostableAlert struct {
+	models.PostableAlert
+	NotificationGroups []string      `json:"notificationGroups,omitempty"`
+	RenotifyInterval   time.Duration `json:"renotifyInterval,omitempty"`
```

Comment:
Why are these two part of the `PostableAlert`? They are the properties of the rule, not the alert instance. They shouldn't be here, i.e part of the `PostableAlert`.